plugin-tray/lxqttray.cpp

```diff
@@ -373,11 +375,19 @@ void LXQtTray::stopTray()
 /************************************************
 
  ************************************************/
+void LXQtTray::onIconDestroyed(QObject * icon)
+{
+    mIcons.removeAll(qobject_cast<TrayIcon*>(icon));
```

Comment:
@palinek 
Could you reproduce the crash RalfJung refers to? How? I mean without this commit.